core/common/src/TextureMapping.ts

```diff
@@ -9,18 +9,38 @@
 import { IndexedPolyfaceVisitor, Matrix3d, Point2d, Point3d, PolyfaceVisitor, Transform, Vector3d } from "@itwin/core-geometry";
 import { RenderTexture } from "./RenderTexture";
 
+/** Defines normal map parameters and texture to use as pattern map when texture is a normal map.
+ * @public
+ */
+export class NormalMap {
+  /** The texture to use as a pattern map. */
```

Comment:
This and `TextureMapping.texture` seem confusing to me.
IMO:
`NormalMap.texture` is the normal map, if defined.
If `NormalMap.texture` is `undefined`, then `TextureMapping.texture` is the normal map and there is no pattern map.
Otherwise, `TextureMapping.texture` is the pattern map.

Reply:
OK, I can change that.  Either way is a bit confusing to me, but I'll try to document it well.

Reply:
Done.  Yes, I like this better.